docs/advanced-scenarios.md

```diff
@@ -152,4 +152,61 @@ Google's `id_token` contains basic identity data like name and user ID, which al
   },
 ```
 
+### Use Feature Toggles in Luigi
+
+There are two possibilities to add feature toggles to the active feature toggles list. On the one hand you can use the core api and on the other hand it is possible to add a feature toggle through url parameters.
+
+#### Set feature toggles to the active feature toggle list through [core api](luigi-core-api.md#featuretoggles)
+```javascript
+  Luigi.featureToggles().setFeatureToggle('ft1');
+```
+
+#### Set feature toggles to the active feature toggle list via url
+```
+  http://localhost:4200/projects/pr1?ft=ft1,ft2
+```
+
+In order to remove a feature toggle from the list you have to use the core api
+
+#### unset feature toggles from the active feature toggle list through core api
+```javascript
+  Luigi.featureToggles().setFeatureToggle('ft1');
```

Comment:
I assume it should be **unsetFeatureToggle** here instead, so I changed it in my suggestion (let me know if it's ok)